cogs/make_member.py

```diff
@@ -266,5 +268,5 @@ async def make_member(self, ctx: TeXBotApplicationContext, group_member_id: str)
         if applicant_role and applicant_role in interaction_member.roles:
             await interaction_member.remove_roles(
                 applicant_role,
-                reason="TeX Bot slash-command: \"/makemember\"",
+                reason='TeX Bot slash-command: "/makemember"',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes